src/main/java/com/uber/jenkins/phabricator/BuildResultProcessor.java

```diff
@@ -71,7 +71,8 @@
 
     public BuildResultProcessor(
             Logger logger, AbstractBuild build, Differential diff, DifferentialClient diffClient,
-            String phid, CodeCoverageMetrics coverageResult, String buildUrl, boolean preserveFormatting) {
+            String phid, CodeCoverageMetrics coverageResult, String buildUrl, boolean preserveFormatting,
+            double coverageFailThreshold) {
```

Comment:
Hmm.. after reading this some more, do you think we can rename this variable to something like `maximumCoverageDecreaseInPercent`?